__tests__/app.test.js

```diff
@@ -128,6 +129,47 @@ describe("GET /api/articles", ()=>{
         })
     })
 
+
+describe("GET /api/articles/:article_id/comments", ()=>{
+    test('200: responds with an array of comments for the given article_id', ()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body})=>{
+            const {comments} = body
+            expect(comments).toBeSortedBy("created_at")
+            comments.forEach((comment)=>{
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: 1
+                })
+            })
+
```

Comment:
When testing with a forEach, it's also important to make sure that the length of comments is not zero, because if it is zero then there's nothing for your for each to iterate though and the tests never have a chance to fail (since they don't even run!)